app/views/feed/contentSelectorPreview.phtml

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+	<body>
+		<?php if ($this->fatalError != '') { ?>
+			<p class="warning"><?= $this->fatalError ?></p>
+		<?php } else { ?>
+			<?php if ($this->selectorSuccess === false) { ?>
+				<p class="warning"><?= _t('feedback.sub.feed.selector_preview.no_result') ?></p>
+				<hr>
+			<?php } ?>
+			<center>
```

Comment:
Please consider replacing (`<center>` is obsolete) https://developer.mozilla.org/docs/Web/HTML/Element/center